File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex25.java
//Importando Scanner
import java.util.Scanner;


public class Ex25 {
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	//vars
	double horas, minutos, horasConvertidas, totalMin, totalSeg;
	
	System.out.println("Conversão de tempo\n\n");
	
	//Entradas	
	System.out.print("Insira a quantidade de horas: ");
	horas = input.nextDouble();
	
	System.out.print("Insira a quantidade de minutos: ");
	minutos = input.nextDouble();
	
	//Processamento
	horasConvertidas = horas * 60;
	totalMin = horasConvertidas + minutos;
	totalSeg = totalMin * 60;
	
	//Saída
	System.out.println("Horas convertidas em minutos: " + horasConvertidas);
	System.out.println("Total de minutos: " + totalMin);
	System.out.println("TOtal de segundos: " + totalSeg);
	
	
	
	
	}

}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex25.c
#include <stdio.h>

main()
{
    //vars
    int h, m, horas, minutos, segundos;

    printf("\t\t\tConversao de Tempo\n\n");

    //Entrada
    printf("Insira a quantidade de horas: ");
    scanf("%d",&h);

    printf("Insira a quantidade de minutos: ");
    scanf("%d",&m);

    //Processamento
    horas = h*60;
    minutos = horas + m;
    segundos = minutos * 60;

    //Saída
    printf("Horas digitadas em minutos: %i min\n",horas);
    printf("Total de minutos: %i min\n",minutos);
    printf("Total de tempo em segundos: %i s\n", segundos);\

}

